src/test/java/com/hydratereminder/command/CommandInvokerTest.java

```diff
@@ -85,6 +85,7 @@ public void shouldReturnNothingWhenIsNotHydrateCommand() {
 
         // when and then
         commandInvoker.invokeCommand(commandToExecute);
+        verify(commandCreator,org.mockito.Mockito.never()).createFrom(any());
```

Comment:
@AbbasSalloum Can you just import `org.mockito.Mockito.never` and just call `never()` here like before?

Reply:
Hello, 

The issue should be resolved.

Anything else I can do ?